<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>equationsGUIClass</class>
 <widget class="QMainWindow" name="equationsGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>equationsGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>315</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListWidget" name="allEquationsList"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="coefficientALabel">
             <property name="text">
              <string>a=</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="coefficientAInput"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="coefficientBLabel">
             <property name="text">
              <string>b=</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="coefficientBInput"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="coefficientCLabel">
             <property name="text">
              <string>c=</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="coefficientCInput"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="updateButton">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="computeSolutionsButton">
           <property name="text">
            <string>Compute solutions</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="firstSolution">
           <property name="text">
            <string>first solution</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="secondSolution">
           <property name="text">
            <string>second solution</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="equationsGUI.qrc"/>
 </resources>
 <connections/>
</ui>
